source/Octopus.Tentacle.Contracts/KubernetesScriptServiceV1/StartKubernetesScriptCommandV1.cs

```diff
@@ -17,7 +17,8 @@ public class StartKubernetesScriptCommandV1
             PodImageConfigurationV1? podImageConfiguration, 
             string? scriptPodServiceAccountName,
             Dictionary<ScriptType, string>? additionalScripts,
-            ScriptFile[]? additionalFiles)
+            ScriptFile[]? additionalFiles,
+            bool isRawScript)
```

Comment:
It's understandable that adding a bool flag to this contract is backwards compatible. However, I would like to float the idea of splitting out the rawscript into its own class, something like "StartKubernetesRawScriptCommand". Having a separate class reduces the complexity in both, as there might not be as many optional inputs as well as fewer bool flags.

Reply:
@kevjt I tend to agree, although contract versioning is a tricky problem. If we did make a new contract, we'd need to probably do a major version of the agent, which then gets tricky.

Due to this, I think it's pragmatic to just add the extra boolean flag as an optional, additional field.

If/when we need a hard, contract breaking version of the agent, we can assess this then.